python/cudf/cudf/core/column/column.py

```diff
@@ -4,13 +4,13 @@
 import warnings
 from numbers import Number
 
+import cupy.binary.elementwise
```

Comment:
Is this used?

Reply:
I'm not sure how this got in here... this likely should have just been `import cupy`